selftests/unit/test_utils_iso9660.py

```diff
@@ -9,6 +9,24 @@
 from avocado.utils import iso9660, process
 
 
+class Capabilities(unittest.TestCase):
+
+    def setUp(self):
+        self.iso_path = os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                                     os.path.pardir, ".data",
+                                                     "sample.iso"))
+
+    def test_common_capabilities(self):
+        none_cap = iso9660.iso9660(self.iso_path)
+        read_cap = iso9660.iso9660(self.iso_path, ['read'])
+        if not (none_cap is None and read_cap is None):
```

Comment:
Instead of checking the ISOs are not `None` (and have any chance to not execute the assertion), isn't better to mock the return of `utils.iso9660.check()` function to ensure this test will always have same behavior in any environment?

Reply:
You mean mocking functions such as `has_pycdlib`, `has_isoinfo`, `has_isoread` and `can_mount`, right? Yes, I think it's a good idea indeed.